modules/iam-dynamic-group/variables.tf

```diff
@@ -47,5 +46,5 @@ variable "policy_compartment_id" {
 
 variable "policy_create" {
```

Comment:
Can we just get rid of policy_create and set policy count to the following instead?:

     count = "${length(var.policy_name) > 0 ? 1 : 0}"